pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -167,7 +167,9 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
             </div>
 
             <div className={styles.addToCatalogusContainer}>
-              <img src={grey} className={styles.componentImage} />
+              <div className={styles.logoContainer}>
+                <img src={_getComponent.data?.embedded?.url?.avatar_url ?? grey} className={styles.componentImage} />
```

Comment:
The default value (having a default value is a good idea) should have a more descriptive name, e.g. `componentPlacholderLogo`.